android_kernel_tools.rc

```diff
@@ -502,9 +508,23 @@ function bootzip()
   signzip "${TmpDir}/${OutFile}.unsigned.zip" "${TmpDir}/${OutFile}";
   cp "${TmpDir}/${OutFile}" "${TargetFile}";
 
+  if [[ ! -z ${PushToDevice} ]]; then
+    echo '';
+    echo '  bootzip: Pushing zip to device!';
+
+    # Waiting to get adb
+    local adb=$(adbcmd);
+    timeout 10 ${adb} 'wait-for-device';
+    printf '  bootzip: ';
+    adb remount;
+
+    timeout 10 ${adb} 'wait-for-device';
+    adb push "${TargetFile}" '/data/media/0/';
```

Comment:
The path is a bit dangerous, not always present or mounted there.
See if /sdcard is fine on your devices, it's the most used path especially in TWRP.